apps/verification-portal/src/lib/components/Toggle.svelte

```diff
@@ -1,7 +1,7 @@
 <!-- Toggle.svelte -->
 <script lang="ts">
   import { Toggle } from 'flowbite-svelte';
-  export let checked: boolean = false;
+  export let checked: boolean | undefined = false;
```

Comment:
It's not a good practice to have var undefined in my perspective. I usually I fix it downstream, so functions always return string for example.